code/modules/research/rdconsole.dm

```diff
@@ -699,9 +694,7 @@ won't update every console in existence) but it's more of a hassle to do. Also,
 				dat += "<div class='statusDisplay'>"
 				if(t_disk.tech_stored[i])
 					var/datum/tech/tech = t_disk.tech_stored[i]
-					dat += "Name: [tech.name]<BR>"
-					dat += "Level: [tech.level]<BR>"
-					dat += "Description: [tech.desc]<BR>"
+					dat += "[tech.name]: [tech.level]"

```

Comment:
Why'd you change this?

Reply:
I dont fully remember. It doesnt fuck anything up does it? My tests seemed ok.

Reply:
Not really, but it removes the tech level description from the console.

Please revert this and we're good to go.

Reply:
So you want the description gone?

Reply:
FUCK IT, SHIP IT